src/libcollections/string.rs

```diff
@@ -1015,8 +1015,7 @@ impl String {
     }
 
     /// Create a draining iterator that removes the specified range in the string
-    /// and yields the removed chars from start to end. The element range is
-    /// removed even if the iterator is not consumed until the end.
```

Comment:
It should be mentioned somewhere in the docs for the method IMO. It's something users have already wondered about, always best to say it explicitly.